frontend/src/components/DeleteCluster.vue

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 -->
 
 <template>
-  <action-icon-dialog
+  <action-button-dialog
```

Comment:
The name should be something like `action-dropdown-menu`. It is not a dialog and in the menu shouldn't occur buttons.